app/lib/common/toolkit/errors/inline_error_button.dart

```diff
@@ -33,29 +34,23 @@ class ActerInlineErrorButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (icon != null) {
-      return _buildWithIcon(context);
-    }
-    return TextButton(
-      onPressed: () async {
-        await ActerErrorDialog.show(
-          context: context,
-          error: error,
-          stack: stack,
-          title: dialogTitle,
-          text: text,
-          textBuilder: textBuilder,
-          onRetryTap: onRetryTap != null
-              ? () {
-                  onRetryTap!();
-                  Navigator.pop(context);
-                }
-              : null,
-          includeBugReportButton: includeBugReportButton,
+    return icon.let((p0) => _buildWithIcon(context, p0)) ??
+        TextButton(
+          onPressed: () => ActerErrorDialog.show(
+            context: context,
+            error: error,
+            stack: stack,
+            title: dialogTitle,
+            text: text,
+            textBuilder: textBuilder,
+            onRetryTap: () => onRetryTap.let((cb) {
+              cb();
+              Navigator.pop(context);
+            }),
```

Comment:
this is incorrect though. if we don't have an `onRetryTap` we shouldn't hand in an empty function but `null`.

```suggestion
            onRetryTap: onRetryTap.let((cb) => () {
              cb();
              Navigator.pop(context);
            }),
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2233/commits/313bf8d1392ea01c0add35d03a523613e590136f